Makefile

```diff
@@ -155,6 +155,7 @@ release: $(KERNEL)
 # Kernel (Linked With Libraries)
 .PHONY: $(KERNEL)
 $(KERNEL):
+	echo $(ROOT)
```

Comment:
This feels like it's left over from debugging

Reply:
Good catch. Thanks.

Reply:
Fixed and rebased.